app/models/theme.rb

```diff
@@ -7,6 +7,16 @@ class Theme < ActiveRecord::Base
   include GlobalPath
 
   BASE_COMPILER_VERSION = 91
+  CORE_THEMES = { "foundation" => -1, "horizon" => -2 }
+  EDITABLE_SYSTEM_FIELDS = %w[
+    child_theme_ids
+    color_scheme_id
```

Comment:
I see that we don't allow edits to color palettes owned by system themes, should we also disallow changing the color palette in the "legacy" color palette system where multiple themes can share a single palette (the `color_scheme_id` column on `themes`)?

Reply:
Hm, probably not. I am not sure. The main reason here was that reimport would revert any colour palette changes, which can be very frustrating to admins.